Build failure with GHC < 8.4

```
Control/Monad/Logger.hs:437:42: error:
    Not in scope: type constructor or class ‘Semigroup’
    |
437 | instance (Applicative m, Semigroup a) => Semigroup (NoLoggingT m a) where
    |                                          ^^^^^^^^^

Control/Monad/Logger.hs:563:42: error:
    Not in scope: type constructor or class ‘Semigroup’
    |
563 | instance (Applicative m, Semigroup a) => Semigroup (WriterLoggingT m a) where
    |                                          ^^^^^^^^^

Control/Monad/Logger.hs:680:42: error:
    Not in scope: type constructor or class ‘Semigroup’
    |
680 | instance (Applicative m, Semigroup a) => Semigroup (LoggingT m a) where
    |                                          ^^^^^^^^^
```

This issue was introduced by https://github.com/snoyberg/monad-logger/pull/34 (CC @jship).

As a Hackage trustee, I have revised the affected version 0.3.37: https://hackage.haskell.org/package/monad-logger-0.3.37/revisions/

--- a/package.yaml
+++ b/package.yaml
@@ -19,7 +19,7 @@ flags:
     default: true
 
 dependencies:
-- base >= 4.8 && <5
+- base >= 4.11 && <5
 - transformers
 - transformers-compat >=0.3
 - text

